client/src/components/Search.js

```diff
@@ -63,6 +59,9 @@ const Search = (props) => {
   const [optionsDeparture, setOptionsDeparture] = useState([]);
   const [optionsArrival, setOptionsArrival] = useState([]);
 
+  const timeoutRef1 = useRef();
+  const timeoutRef2 = useRef();
+
   const handleSubmit = async () => {
     const departureLocationId = await getCityId(state.departureCity);
```

Comment:
Nice abstraction by using functions written in ../utils/skyscanner. Makes this search component leaner.